app/cypress/integration/cif/project-revision/index.spec.js

```diff
@@ -528,7 +528,19 @@ describe("the new project page", () => {
     cy.findByText(/submit change/i).click();
     cy.findByText(/review and submit information/i).click();
     cy.findByText(/review and submit project/i).should("exist");
-    cy.get("textarea").clear().type("foo");
+    cy.get("textarea").click().type("foo");
+
+    cy.injectAxe();
+    // Temporarily disable axe for dl accessibility checks
```

Comment:
What's the reason for disabling? Is it the div inside a table issue?

Reply:
It's because we have accessibility issues with `dl` and `dt` elements as stated in tech debt card [here](https://github.com/bcgov/cas-cif/issues/563)
We need to set a rule for that until we resolve that issue.